traffic_ops/testing/api/v3/jobs_test.go

```diff
@@ -82,6 +83,28 @@ func CreateTestInvalidationJobs(t *testing.T) {
 	}
 }
 
+func CreateInvalidJob(t *testing.T) {
+	toDSes, _, err := TOSession.GetDeliveryServicesNullable()
+	if err != nil {
+		t.Fatalf("cannot GET Delivery Services: %v - %v", err, toDSes)
+	}
+	dsNameIDs := map[string]int64{}
+	for _, ds := range toDSes {
+		dsNameIDs[*ds.XMLID] = int64(*ds.ID)
+	}
+
+	job := testData.InvalidationJobs[0]
+	_, ok := dsNameIDs[(*job.DeliveryService).(string)]
+	if !ok {
+		t.Fatalf("can't create test data job: delivery service '%v' not found in Traffic Ops", job.DeliveryService)
+	}
+	tooHigh := interface{}(2161)
```

Comment:
2160 is a well-known TTL, but this magic number should still be factored into `x * y * [...]`

Reply:
done in bae7ca548739e0d9fe501afba8b2c2559b7843f9